src/poliastro/czml/extract_czml.py

```diff
@@ -403,29 +187,43 @@ def add_ground_station(
                 "Invalid coordinates. Coordinates must be of the form [u, v] where u, v are astropy units"
             )
 
-        self._init_ground_station_packet_(self.gs_n, pos)
-        self._change_ground_station_params_(
-            id_name=id_name,
-            id_description=id_description,
-            label_fill_color=label_fill_color,
-            label_font=label_font,
-            label_outline_color=label_outline_color,
-            label_text=label_text,
-            label_show=label_show,
+        pckt = Packet(
+            id="GS" + str(self.gs_n),
+            description=id_description,
+            availability=self.start_epoch.value + "/" + self.end_epoch.value,
+            position=Position(cartesian=pos),
+            label=Label(
+                show=label_show,
+                text=label_text,
+                font=label_font if label_font is not None else "11pt Lucida Console",
```

Comment:
Labels have a default font of `30px sans-serif` https://github.com/AnalyticalGraphicsInc/czml-writer/wiki/Label#properties do we need to set it here?